app/src/main/java/io/homeassistant/companion/android/settings/shortcuts/views/ManageShortcutsView.kt

```diff
@@ -0,0 +1,447 @@
+                    label = { Text(stringResource(id = R.string.lovelace_view_dashboard)) },
+                    modifier = Modifier.padding(top = 16.dp, bottom = 16.dp)
+                )
+            } else {
+                Text(
+                    text = stringResource(id = R.string.entity_id),
+                    fontSize = 15.sp
+                )
+                OutlinedButton(onClick = { expandedEntity = true }) {
+                    Text(
+                        text = when (index) {
+                            1 -> viewModel.shortcutPath1.value
+                            2 -> viewModel.shortcutPath2.value
+                            3 -> viewModel.shortcutPath3.value
+                            4 -> viewModel.shortcutPath4.value
+                            5 -> viewModel.shortcutPath5.value
+                            6 -> viewModel.shortcutPathPinned.value
+                            else -> viewModel.shortcutPath1.value
+                        }
+                    )
+                }
+
+                DropdownMenu(expanded = expandedEntity, onDismissRequest = { expandedEntity = false }) {
+                    val sortedEntities = viewModel.entities.values.sortedBy { it.entityId }
+                    for (item in sortedEntities) {
+                        DropdownMenuItem(onClick = {
+                            when (index) {
+                                1 -> viewModel.shortcutPath1.value = "entityId:${item.entityId}"
+                                2 -> viewModel.shortcutPath2.value = "entityId:${item.entityId}"
+                                3 -> viewModel.shortcutPath3.value = "entityId:${item.entityId}"
+                                4 -> viewModel.shortcutPath4.value = "entityId:${item.entityId}"
+                                5 -> viewModel.shortcutPath5.value = "entityId:${item.entityId}"
+                                6 -> viewModel.shortcutPathPinned.value = "entityId:${item.entityId}"
+                            }
+                            expandedEntity = false
+                        }) {
+                            Text(text = item.entityId, fontSize = 15.sp)
+                        }
+                    }
+                }
+            }
+            for (item in viewModel.dynamicShortcuts.value) {
+                if (item.id == shortcutId) {
+                    when (index) {
+                        1 -> viewModel.deleteShortcut1.value = true
+                        2 -> viewModel.deleteShortcut2.value = true
+                        3 -> viewModel.deleteShortcut3.value = true
+                        4 -> viewModel.deleteShortcut4.value = true
+                        5 -> viewModel.deleteShortcut5.value = true
+                    }
+                }
+            }
+            Button(
+                onClick = {
+                    when (index) {
+                        1 -> {
+                            if (viewModel.deleteShortcut1.value)

```

Comment:
Why is this only shown for shortcut 1, and not for all shortcuts?

Reply:
totally missed this one 😅